tests/utils.py

```diff
@@ -69,7 +69,7 @@ def get_fqid(value: str) -> FullQualifiedId:
     """
     Returns a FullQualifiedId parsed from the given value.
     """
-    collection, id = value.split(KEYSEPARATOR)
+    collection, id, *catch_trailing_slash = value.split(KEYSEPARATOR)
```

Comment:
Please do not catch more slashes here. Instead we should check an raise an error if someone tries to use an invalid value here.